aait/backend/group-2A/Application/Contracts/Persistance/INotificationRepository.cs

```diff
@@ -4,5 +4,8 @@ namespace Application.Contracts.Persistance;
 
 public interface INotificationRepository{
      Task<List<Notification>> GetNotifications(int id);
+     Task<Notification> GetNotification(int notifiactionId);
      Task  AddNotification(Notification notify);
```

Comment:
I will be better if you use the Generic Add() method, why do you need to define custom method.